main.py

```diff
@@ -20,6 +20,16 @@ def suutiit_zinju():
   dati = request.json
   chats.pieraksti_zinju(dati)
   return chats.lasi()
+  
+@app.route('/chats/get_mood')
+def sanjemt_zinju():
```

Comment:
Šis jau izskatās pēc "saņemt garastāvoklus"
varētu action: "/garastavokli/sanemt/" un metodi:
```python
def sanemt_garastavoklus():
```
Bet mums tas neko nedos, jo potenciāli 2 cilvēki aizsūta "A" un "B", mēs nevarēsim identificēt ar esošo risinājumu kuram kurš ir.